src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/set_interior_vehicle_data_request.cc

```diff
@@ -202,6 +203,30 @@ bool CheckLightNameByCapabilities(
   return false;
 }
 
+bool CheckRadioBandByCapabilities(
```

Comment:
@LitvinenkoIra please add doxygen documentation. Specify what doe it means true or false in return status, and in what cases.